bigchaindb/utils.py

```diff
@@ -167,3 +167,15 @@ def run(self, instance):
 
         self.stack = []
         return last
+
+
+# Load Tendermint's public and private key from the file path
+def load_node_key(path):
+    import json
+    from bigchaindb.tendermint_utils import key_from_base64
+    from bigchaindb.common.crypto import key_pair_from_ed25519_key
```

Comment:
Why are the imports done inside the function?

Reply:
Moved to top